### DISCUSSION TITLE
posts/661

### DISCUSSION THREAD

**Discussion by giscus** [kakaotech/comments#26] [Announcements]
# posts/661

안녕하세요, 카카오의 자체 언어모델을 개발하고 있는 Language Model T...

https://tech.kakao.com/posts/661

**Comment by bzantium** [comment]
유익한 내용이네요!

**Comment by aiqwe** [comment]
좋은 글 감사합니다 :D

**Comment by jujbob** [comment]
한-영 언어모델 Bllossom 모델 개발을 하고있는 MLP 연구실 입니다!

정말 양질의 연구개발 내용을 공유해 주셔서 감사드립니다. 

특히 (돈 많이 태우셨을텐데) 이런 연구 내용을 공유할 수있도록 의사결정한 Kakao 연구팀의 독립성에 감탄 했습니다 ㅎㅎ. 제가 만나본 많은 한국 기업들은 숨기느라 바바빴거든요. 기회가 된다면 저희가 찾아낸 한국어 llm 내용 관련해 이야기해봐도 정말 재미 있을것 같습니다. 앞으로도 좋은 자료 부탁합니다 :)

**Reply by seopbo** [reply]
감사합니다!
